CONTRIBUTING.md

```diff
@@ -26,6 +26,14 @@ If you run `yarn start` and encounter the following error, try rerunning `yarn s
 >  NX   ENOENT: no such file or directory, open 'storybook/code/node_modules/nx/package.json'
 ```
 
+In case you come back as a contributor after some time, you might have to clean your local files before `yarn start`:
+
+```sh
+git clean -dx --dry-run
+```
+
+Make sure you have no uncommitted changes you might lose and rerun the above with the `--force` flag.
```

Comment:
@bodograumann, we have a bit of room for improvement here. I was thinking we could adjust this text to be something more along the lines of:

````
If you are a Storybook contributor and still experiencing issues, it is recommended that you verify your local Storybook instance for any changes that you may have made locally. To do this, you can use the following command:

```sh
git clean -dx --dry-run
```

By executing this command, you will be able to see which untracked or ignored files and directories will be removed from your working directory if you run it with the `--force` flag. Before running the command with the `--force` flag, ensure you do not have any local changes you want to keep.
````

If you're ok with it, can you adjust it?

Reply:
Thanks for getting back to me @jonniebigodes .
This is not quite what I had in mind.
While it is important to not throw away uncommited local changes, normally I would not expect anybody to have manually created files that need to be cleaned up.
Rather (at least in my case) all the things automatically created e.g. when running `yarn start` and running the dev toolchain in the repo seem to become stale after a while. Then simply fast-forwarding the `next` branch and doing `yarn start` again fails.
Those files are ignored by git, so by using `git clean` it is possible to start from a clean slate, just as if the repo was freshly checked out.

Reply:
@bodograumann I think the changes @jonniebigodes proposed are more specific about what is actually being done, but not going against what you had in mind, unless I am missing your point.

Reply:
Hi @Marklb . I mainly wouldn't write ”any changes that you may have made locally.”, because that suggests to the reader those changes could cause problems, while it is actually more about automatically generated files.

Reply:
I see that point. I was considering auto generated files to still be changes that I made locally, since I performed the action that generated them locally. Maybe it would be better to say something about *verifying that you don't have untracked files that should not be deleted*, instead of calling them changes?

Reply:
I have updated the text based on the suggestions, with some slight changes. Please have another look.